.github/workflows/publish-helm-charts.yaml

```diff
@@ -0,0 +1,37 @@
+name: Publish Helm charts
+
+on:
+  release:
+    types:
+      - published
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+        contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: main
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+
+      - name: Run chart publisher script
+        run: |
+          ./scripts/build/helm-publisher.sh \
+          ${{ join(github.event.release.assets.*.browser_download_url, ' ') }} \
+          ${{ github.event.release.tag_name }}
```

Comment:
Hmm, does this work as expected as you have `browser_download_url="$1"` in the publisher script? Do we really need the release `tag_name` for something? Wouldn't it be simpler to drop that and only have the urls as cmdline args?